src/Microsoft.SqlTools.ServiceLayer/ExecutionPlan/Contracts/ExecutionPlanGraph.cs

```diff
@@ -74,6 +74,10 @@ public class ExecutionPlanNode
         public string[] Subtext { get; set; }
         public List<ExecutionPlanNode> Children { get; set; }
         public List<ExecutionPlanEdges> Edges { get; set; }
+        /// <summary>
+        /// Add an overlay icon to the node like warnings and parallelism
+        /// </summary>
+        public List<ExecutionPlanNodeOverlay> OverlayIcons { get; set; }
```

Comment:
Generally the names should match, so I'd expect either the property to be named "Overlays" or the class to be "ExecutionPlanNodeOverlayIcon"

But also - maybe "Badge" would be a better term here - that's usually what is used to indicate the status icons (like the connected status on servers)